packages/api-form-builder/src/plugins/triggers/triggerHandlers.ts

```diff
@@ -1,43 +1,135 @@
 import got from "got";
+import { SmtpTransportConfig, createSmtpTransport } from "@webiny/api-mailer";
 import { FbFormTriggerHandlerPlugin } from "~/types";
 
-const plugin: FbFormTriggerHandlerPlugin = {
-    type: "form-trigger-handler",
-    name: "form-trigger-handler-webhook",
-    trigger: "webhook",
-    async handle({ trigger, data, addLog }) {
-        const urls = trigger && trigger.urls;
-        if (!urls || Array.isArray(urls) === false) {
-            return;
+const MAILER_CONFIG: SmtpTransportConfig = {
+    host: process.env.WEBINY_MAILER_HOST,
+    auth: {
+        user: process.env.WEBINY_MAILER_USER,
+        pass: process.env.WEBINY_MAILER_PASSWORD
+    }
+};
+
+const plugins: FbFormTriggerHandlerPlugin[] = [
+    {
+        type: "form-trigger-handler",
+        name: "form-trigger-handler-webhook",
+        trigger: "webhook",
+        async handle({ trigger, data, addLog }) {
+            const urls = trigger && trigger.urls;
+            if (!urls || Array.isArray(urls) === false) {
+                return;
+            }
+            for (const url of urls) {
+                /**
+                 * Could be executed without awaiting the end result of the trigger? Not sure how it would
+                 * work in Lambda, so for now, let's await the result of the request, and update form submission
+                 * logs accordingly.
+                 */
+                try {
+                    const response = await got(url, {
+                        method: "post",
+                        json: true,
+                        body: data
+                    });
+
+                    addLog({
+                        type: "success",
+                        message: `Successfully sent a POST request to ${url}`,
+                        data: {
+                            response: response.body
+                        }
+                    });
+                } catch (e) {
+                    addLog({
+                        type: "warning",
+                        message: `POST request to ${url} failed: ${e.message}`
+                    });
+                }
+            }
         }
-        for (const url of urls) {
-            /**
-             * Could be executed without awaiting the end result of the trigger? Not sure how it would
-             * work in Lambda, so for now, let's await the result of the request, and update form submission
-             * logs accordingly.
-             */
+    },
+    {
+        type: "form-trigger-handler",
+        name: "form-trigger-handler-email-notification",
+        trigger: "email-notification",
+        async handle({ trigger, data, form, addLog }) {
+            const email = trigger && trigger.email;
+            if (!email) {
+                return;
+            }
+
+            const mailer = createSmtpTransport(MAILER_CONFIG);
```

Comment:
Just checked with the team.

Mailer should be used via the `context` object, and not directly via `createSmtpTransport`.

Example usage: https://www.webiny.com/docs/overview/features/mailer#usage

Thanks!